pkg/backend/display/options.go

```diff
@@ -57,6 +57,9 @@ type Options struct {
 	SuppressTimings        bool                // true to suppress displaying timings of resource actions
 	SuppressProgress       bool                // true to suppress displaying progress spinner.
 	ShowLinkToCopilot      bool                // true to display a 'explainFailure' link to Copilot.
+	ShowCopilotSummary     bool                // true to display a Copilot summary.
+	CopilotSummaryModel    string              // the Copilot summary model to use.
+	CopilotSummaryMaxLen   int                 // the maximum length of the Copilot summary.
```

Comment:
Probably worth adding "Defaults to ..." to these comments?

Reply:
And I guess "Ignored if `ShowCopilotSummary` is `false`"? Not sure also if there's valuable commentary on `ShowLinkToCopilot`. TBH feels like we're getting to the point where we probably want `CopilotOptions` as a struct that has all this stuff in it?